playback/src/player.rs

```diff
@@ -1111,7 +1127,14 @@ impl PlayerTrackLoader {
             };
```

Comment:
FYI, this needs to be updated to use a `Hint` also. Currently doesn't compile with `passthrough-decoder` enabled.